core/pagination.py

```diff
@@ -0,0 +1,5 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class PageNumberPagination(PageNumberPagination):
+    page_size_query_param = 'limit'
```

Comment:
A more descriptive name would be `page_size`. Using `limit` could lead to confusion; since the same value is the default query parameter for [`rest_framework.pagination.LimitOffsetPagination`](http://www.django-rest-framework.org/api-guide/pagination/#limitoffsetpagination)

Reply:
We can still limit the number of items per page my setting
```python
page_size = 10
```
The effect of this would be
 - All API responses return 10 items per page
- API requests that require the entire response can specify a greater `page_size` param i.e `?page_size=999`

This approach ensures that the more frequent API calls i.e `/api/v1/assets` do not fetch all the data while allowing for the FE to be explicit when requesting for all data.

Reply:
I think we could have a different way to specify all data, that a number.. if it gets to a point where we have eg more that 999 assets it'd still be paginated.
Is it possible to add another parameter? something like `paginated=false` which defaults to `true` if not specified?